docs/building-apps/network-features/vetkeys/introduction.mdx

```diff
@@ -10,89 +10,42 @@ import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
-
-In a private auction dapp, users can submit bids that are IBE-encrypted under an identifier so that at the end of the auction, the dapp can decrypt all bids with a single vetKey evaluation. A similar technique can be used to prevent front-running, also known as miner-extracted value (MEV), on a decentralized exchange (DEX). Users submit their IBE-encrypted transactions under a predictable batch identifier. The DEX orders the transactions in encrypted form and, when all transactions for a particular batch have been ordered, triggers the recovery of the decryption key for that batch and executes the decrypted transactions in the fixed order. Note that all of the symmetric-encryption use cases listed above can be modified to encrypt using an IBE instead of a symmetric-key encryption, thereby eliminating the need to perform a vetKey derivation for encryption. Decryption, of course, still requires retrieving a vetKey.
-
-#### Advanced timelocks
+## vetKeys use cases and applications
 
-Anyone can IBE-encrypt their message using some more complex condition as identity, while a dapp lets anyone who can satisfy the condition (or when the condition is satisfied on its own) to obtain the respective vetKey. This enables quite interesting use cases, as it enables one to encrypt to any verifiable future event, e.g., the price of a stock going above or below a certain level, information escrow, or break-the-glass policies.
+The availability of vetKeys allows for a series of applications, including but not limited to those covered in the following sections.
 
-### [Externally verifiable randomness](/docs/building-apps/network-features/vetkeys/advanced/verifiable-randomness)
+### Distributed key management service (DKMS)
 
-Because of their uniqueness, vetKeys can also act as a Verifiable Random Function (VRF). Trusted, verifiable randomness is important for applications such as trustless online lotteries and casinos, fair decentralized games (GameFi), and selecting random features for non-fungible tokens (NFTs).
+DKMS is an abstraction on top of vetKeys, designed to simplify obtaining and managing access permissions to vetKeys. It allows users to securely generate, manage, and share cryptographic keys using vetKeys and ICP. These vetKeys can then be used for encryption, decryption, or data signing. Read [here](/docs/building-dapps/network-features/vetkeys/dkms) about how DKMS can be instantiated in your dapp.
 
-Note that if your dapp does not need external verification (this is normally the case for dapps with public code that are running solely on ICP and no other blockchains), it is more efficient and much cheaper to use secure randomness on ICP that is not externally verifiable, see TODO:LINK.
+### Encrypted onchain storage
 
-## vetKeys API
+Encrypted onchain storage is another abstraction on top of DKMS that enables data encryption using vetKeys in the frontend, storage of the encrypted data, and management of access permissions. 
+Its goal is twofold:
```

Comment:
Think of this section as trying to explain what is possible with vetkeys, not advertising the example/tutorial or a specific instantiation (we should still link to the example).